sycl/source/detail/pi.cpp

```diff
@@ -16,9 +16,28 @@
 #include <stddef.h>
 #include <string>
 
+#ifdef XPTI_ENABLE_INSTRUMENTATION
+// Include the headers necessary for emitting
+// traces using the trace framework
+#include "xpti_trace_framework.h"
+// Stream name being used for traces generated
+// from the SYCL runtime
+#define PICALL_STREAM_NAME  "sycl.pi"
```

Comment:
Maybe constexpr const char * ?

Reply:
And please, do not define things in global namespace, there is cl::sycl::detail namespace for internal things.

Reply:
Done!